controllers/vrg_recipe.go

```diff
@@ -54,7 +54,29 @@ func captureWorkflowDefault(vrg ramen.VolumeReplicationGroup, ramenConfig ramen.
 	return captureSpecs
 }
 
-func recoverWorkflowDefault() []kubeobjects.RecoverSpec { return []kubeobjects.RecoverSpec{{}} }
+func recoverWorkflowDefault(vrg ramen.VolumeReplicationGroup, ramenConfig ramen.RamenConfig) []kubeobjects.RecoverSpec {
+	namespaces := []string{vrg.Namespace}
+
+	if vrg.Namespace == RamenOperandsNamespace(ramenConfig) {
+		namespaces = *vrg.Spec.ProtectedNamespaces
+	}
+
+	recoverSpecs := []kubeobjects.RecoverSpec{
+		{
+			Spec: kubeobjects.Spec{
+				KubeResourcesSpec: kubeobjects.KubeResourcesSpec{
+					IncludedNamespaces: namespaces,
+				},
+			},
+		},
+	}
+
+	if vrg.Spec.KubeObjectProtection.KubeObjectSelector != nil {
+		recoverSpecs[0].Spec.LabelSelector = vrg.Spec.KubeObjectProtection.KubeObjectSelector
```

Comment:
(nit) This line can be part of the spec creation and moved to line 70 and remove the `if` block. Either way, the `LabelSelector` will be either `nil` or a value.

Reply:
Done.